lib/android_apk/signature_verifier.rb

```diff
@@ -8,7 +8,7 @@ module SignatureVerifier
     # [min_sdk(inclusive), max_sdk(inclusive)]
     SCHEME_CASE_INPUTS = [
       [
-        8,
+        9,
```

Comment:
This was a bug but not important because this sdk version rarely used as min sdk in the world.

![ss 2023-11-07 at 15 33 08](https://github.com/DeployGate/android_apk/assets/4340693/b188c0cc-b12b-418f-8e7c-14244542192e)